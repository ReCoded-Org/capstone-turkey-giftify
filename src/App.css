.mainContainer {
  padding: 0 !important;
  background-color: #0077b6;
}