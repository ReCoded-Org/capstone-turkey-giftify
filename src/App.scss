@import "~bootstrap/scss/bootstrap";

.container-fluid {
    padding: 0 !important;
    background-color: #0077B6;

}