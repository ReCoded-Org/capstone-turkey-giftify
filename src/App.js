import React from "react";
import "bootstrap/dist/css/bootstrap.min.css";
import { Container } from "react-bootstrap";
import { Route } from "react-router";
import { BrowserRouter as Router } from "react-router-dom";
import "./App.css";
import Footer from "./components/Footer/Footer";
import Navbar from "./components/Navbar/Navbar";
import HomePage from "./containers/HomePage/HomePage";
import RequestItem from "./containers/RequestItem/RequestItem";

function App() {
  return (
    <>
      <Container fluid className="mainContainer">
        <Router>
          <Route path="/" component={Navbar} />
          <Route exact path="/" component={HomePage} />
          <Route exact path="/request-item/:id" component={RequestItem} />
          {/* <Route path="/requests" component={Requests} />
          <Route path="/donations" component={Donations} />
          <Route path="/about" component={About} />
          <Route path="/contactUs" component={ContactUs} /> */}
          <Route path="/" component={Footer} />
        </Router>
      </Container>
    </>
  );
}

export default App;
