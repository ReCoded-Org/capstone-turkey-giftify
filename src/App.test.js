import React from "react";
import { Provider } from "react-redux";
import { cleanup } from "@testing-library/react";
import renderer from "react-test-renderer";
import store from "./redux/store";
import App from "./App";

afterEach(cleanup);

it("App rendered correctly", () => {
  const tree = renderer
    .create(
      <Provider store={store}>
        <App />
      </Provider>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
