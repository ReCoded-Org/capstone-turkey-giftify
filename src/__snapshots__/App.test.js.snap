// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="App"
>
  <div
    id="navbar"
  >
    <div
      id="navbarLogo"
    >
      <a
        href="/"
        onClick={[Function]}
      >
        LOGO
      </a>
    </div>
    <div
      id="navbarLinks"
    >
      <a
        className="eachLinksInNavbar"
        href="/"
        onClick={[Function]}
      >
        Home
      </a>
      <a
        className="eachLinksInNavbar"
        href="/#search"
        onClick={[Function]}
      >
        Search
      </a>
      <a
        className="eachLinksInNavbar"
        href="/requests"
        onClick={[Function]}
      >
        Requests
      </a>
      <a
        className="eachLinksInNavbar"
        href="/donations"
        onClick={[Function]}
      >
        Donations
      </a>
      <a
        className="eachLinksInNavbar"
        href="/about"
        onClick={[Function]}
      >
        About
      </a>
      <a
        className="eachLinksInNavbar"
        href="/contactUs"
        onClick={[Function]}
      >
        Contact Us
      </a>
      <div
        className="dropdown"
      >
        <button
          aria-expanded={false}
          aria-haspopup={true}
          className="dropdown-toggle btn btn-primary"
          disabled={false}
          id="dropdown-basic-button"
          onClick={[Function]}
          type="button"
        >
          Language
        </button>
      </div>
    </div>
    <div
      id="navbarProfile"
    >
      <a
        href="/logIn"
        onClick={[Function]}
      >
        <button
          className="btn btn-primary"
          disabled={false}
          style={
            Object {
              "borderRadius": "0.75rem",
              "marginBottom": "5%",
            }
          }
          type="button"
        >
          Log In
        </button>
      </a>
      <a
        href="/signUp"
        onClick={[Function]}
      >
        <button
          className="btn btn-primary"
          disabled={false}
          style={
            Object {
              "borderRadius": "0.75rem",
              "marginBottom": "5%",
            }
          }
          type="button"
        >
          Sign Up
        </button>
      </a>
    </div>
  </div>
</div>
`;
